src/main/cpp/commands/LatchDisengage.cpp

```diff
@@ -0,0 +1,25 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/LatchDisengage.h"
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.
+// For more information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+LatchDisengage::LatchDisengage() {
```

Comment:
See previous comments about subsystems and commands. Commands need subsystems by reference

```suggestion
LatchDisengage::LatchDisengage(Shooter& shooter) {
```